sys/tests/README.md

```diff
@@ -1,7 +1,6 @@
-Test infrastructure
----
+# Test infrastructure
```

Comment:
Why tests for `SIGPIPE` require changes in README?

Reply:
Right. Please move it to another PR. As you write new tests I guess you'll be in position to improve `tests/README.md` several times.

Reply:
Done, #1143 was born